fms_sdg/databuilders/simple/generate.py

```diff
@@ -32,7 +32,7 @@ def __init__(
         self,
         *args: Any,
         num_prompt_instructions: Optional[int] = 2,
-        prompt_file_path: Optional[str] = None,
+        prompt_file_path: Optional[str] = "prompt.txt",
         request_batch_size: Optional[int] = 5,
```

Comment:
As we have valid defaults why not?
```suggestion
        prompt_file_path: str = "prompt.txt",
        request_batch_size: int = 5,
```